### PR TITLE
Add support for PatientCharacteristic QDM datatype

### PR DESCRIPTION
**Pull request by mayerm94**
https://jira.mitre.org/browse/BONNIE-632
https://oncprojectracking.healthit.gov/support/browse/BONNIE-266

### CODE REVIEW

File: app/assets/javascripts/datatypes/patient_characteristic.js.coffee [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+###
+Used to represent a sex.
+###
+class CQL_QDM.PatientCharacteristic extends CQL_QDM.QDMDatatype
+  ###
+  @param {Object} patient - the HDS patient object to use
+  ###
+  constructor: (@entry) ->
+    super @entry
+    debugger
```
Comment (holmesie) [inline app/assets/javascripts/datatypes/patient_characteristic.js.coffee]: Might want to remove that

Comment (mayerm94) [inline app/assets/javascripts/datatypes/patient_characteristic.js.coffee]: oopsie doopsie

**Comment by mayerm94** [comment]
Check out https://ecqi.healthit.gov/system/files/QDM_5_3_2017June_508.pdf there is a datatype of just patient characteristic that has a code and an author date time. The spec specifies that the patient characteristic datatype is to be used to "document a characteristic of the patient not represented by one of the more specific Individual Characteristic datatypes." So it would make sense to use the PatientCharacteristicSex datatype. I talked to Paul about this because I had the same thought as you @holmesie , and he said that the MAT probably made a datacriteria for gender using PatientCharacteristic because they were using a valueset for Gender that is different than the standard used SDE Sex.

**Comment by mayerm94** [comment]
[Telligen_645_v5_2_Artifacts.zip](https://github.com/projecttacoma/cql_qdm_patientapi/files/1075364/Telligen_645_v5_2_Artifacts.zip)
Check out in the HQMF/ELM the difference between PatientCharacteristic and PatientCharacteristicSex

**Comment by holmesie** [comment]
Okay that makes sense; I'm happy with this change as long as you modify the class comment to reflect that definition from the spec.

**Pull request merged**